sdks/java/core/src/main/java/org/apache/beam/sdk/options/PortablePipelineOptions.java

```diff
@@ -103,4 +103,32 @@
   String getOutputExecutablePath();
 
   void setOutputExecutablePath(String outputExecutablePath);
+
+  @Description(
+      "Options for configuring the environment of portable workers. Recognized options depend on the value of defaultEnvironmentType:\n"
```

Comment:
Mention that this will be used to configure default environment of the pipeline SDK but not the external SDKs of any cross-language pipelines used by the pipeline.

Reply:
Done.